test/aspect/aspect.bzl

```diff
@@ -45,8 +45,6 @@ def _rule_impl(ctx):
             "@io_bazel_rules_scala//specs2:specs2",
             "@scala//:scala-xml",
             "@scala//:scala-parser-combinators",
-            "@scala//:scala-library",
```

Comment:
@chaoren does this mandate changes in the plugin? So that it will take these dependencies from the toolchain?